2018/001-containers-tagging-strategy.md

```diff
@@ -0,0 +1,189 @@
+﻿# Containers Tagging Strategy
+
+| Field  | Value      |
+|:-------|:-----------|
+| Status | Draft      |
+| Date   | 2018-08-8  |
+
+## Introduction
+
+As SUSE starts delivering software shipped in containers accessible
+from the [SUSE container registry](https://registry.suse.de). Images within the
```

Comment:
That's the internal registry, the final one is going to be `registry.suse.com`.

Reply:
Yes, I know, just added it here because this link is browsable from SUSE network. We can remove the link if this is confusing. The registry.suse.com to me leads to a `Connection refused` error

Reply:
This is kubic-project - so maybe use registry.opensuse.org?